### ISSUE TITLE
including "*" doesn't fail, yet also doesn't include relations

### ISSUE THREAD

**Issue by ldiebold** [tailflow/laravel-orion#198]
We're trying to do some quick prototyping and want to whitelist all relationships.

A request for data works with the following, however the relationship is not loaded:
```php
<?php

namespace App\Http\Controllers\Api;

use App\Models\Tank;
use Orion\Http\Controllers\Controller;

class TankController extends Controller
{
    protected $model = Tank::class;

    public function includes(): array
    {
        return ['*'];
    }
}
```

Request looks like this:
```
http://localhost/api/tanks?include=org
```

And if I do the following, everything works as expected (org is loaded in the response):
```php
<?php

namespace App\Http\Controllers\Api;

use App\Models\Tank;
use Orion\Http\Controllers\Controller;

class TankController extends Controller
{
    protected $model = Tank::class;

    public function includes(): array
    {
        return ['org'];
    }
}
```

Let me know if you need any more information, or a reproduction!

**Comment by alexzarbn** [comment]
Hi @ldiebold,

Thank you for reporting the issue - I will look into it.

**Comment by alexzarbn** [comment]
Fixed in [v2.17.4](https://github.com/tailflow/laravel-orion/releases/tag/2.17.4)
